src/css/main.less

```diff
@@ -3,10 +3,13 @@
 @import (inline) "vendor/tooltip.less";
 
 @font-face {
-  font-family: 'Open Sans';
+  font-family: 'Source Sans Pro', 'Helvetica Neue', Helvetica, Arial, sans-serif;
   font-style: normal;
   font-weight: 300;
-  src: local('Open Sans Light'), local('OpenSans-Light'), url(https://themes.googleusercontent.com/static/fonts/opensans/v8/DXI1ORHCpsQm3Vp6mXoaTaRDOzjiPcYnFooOUGCOsRk.woff) format('woff');
+  src: local('Source Sans Pro Light'),
+       local('SourceSansPro-Light'),
+       url('http://themes.googleusercontent.com/static/fonts/sourcesanspro/v4/toadOcfmlt9b38dHJxOBGNbE_oMaV8t2eFeISPpzbdE.woff') format('woff'),
```

Comment:
Let's remove this.
Marathon is used in intranets and this request opens a window into the wide world to Google without the knowledge of the user. Really bad in terms of privacy.
I think loading it from us isn't slower, probably faster then from Google.
I don't rely on a cache..